src/graphql/error-handler.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQLFormattedError } from "graphql";
+import { unwrapResolverError } from "@apollo/server/errors";
+
+export class CustomError extends Error {
+  code: number;
+  additionalInfo: string;
+  constructor(code: number, message: string, additionalInfo?: string) {
+    super(message);
+    this.code = code;
+    this.additionalInfo = additionalInfo || "No additional info";
```

Comment:
```suggestion
    this.additionalInfo = additionalInfo ?? "No additional info";
```

Ou outra opção seria vc deixar esse campo não obirgatório tbm e enviar nulo caso não tenha necessidade.